src/app/models/submission.server.model.ts

```diff
@@ -156,6 +133,28 @@ const encryptSubmissionSchema = new Schema<IEncryptedSubmissionSchema>({
   webhookResponses: [webhookResponseSchema],
 })
 
+/**
+ * Returns an object which represents the encrypted submission
+ * which will be posted to the webhook URL.
+ */
+
+encryptSubmissionSchema.methods.getWebhookView = function (
+  this: ISubmissionSchema,
+): WebhookView {
+  const webhookData: WebhookData = {
+    formId: String(this.form),
+    submissionId: String(this._id),
+    encryptedContent: this.encryptedContent,
+    verifiedContent: this.verifiedContent,
+    version: this.version,
+    created: this.created,
+  }
+
+  return {
+    data: webhookData,
+  }
+}
+
 type IEmailSubmissionModel = Model<IEmailSubmissionSchema>
```

Comment:
Might want to move the types to `src/types` for consistency

Reply:
will do